### ISSUE TITLE
[request] @roots/sage: include `.blade.php` in compilation

### ISSUE THREAD

**Issue by erip2** [roots/bud#2006]
### Agreement

- [X] This is not a duplicate of an existing issue
- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

### Describe the issue

I'm trying to show images in the webp format if supported.
When trying to use it in the CSS, everything works fine.

```
.test {
  background-image: url('images/my-image.jpg?as=webp');
}
```

After running the build script, the webp version is created and is displayed if the browser supports it.

When trying it to use in the blade template, like this:

```
<picture>
      <source srcset="@asset('images/my-image.jpg?as=webp')" type="image/webp" media="(max-width: 767px)">
      <source srcset="@asset('images/my-image.jpg')" type="image/jpeg" media="(max-width: 767px)">

      <img src="@asset('images/my-image.jpg')" alt="My Image">
    </picture>
```

The webp format is not generated by the extension and in the front-end, the new URL is generated as is, without hashes.

I tried this in a freshly new Sage theme to be sure that this wasn't an issue on my project. 

I'm expecting this to work since I saw this use on the `image-minimizer-webpack-plugin` documentation [here](https://github.com/webpack-contrib/image-minimizer-webpack-plugin#query-parameters-only-squoosh-and-sharp-currently).

### Expected Behavior

The image to be converted to `.webp` and be shown on the HTML.

### Actual Behavior

The image is not shown at all.

### Steps To Reproduce

1. Create a new Sage theme
2. Add `bud-imagemin`
3. Add an image to `images` folder 
4. Reference that image in the blade and add the webp generator in the end: `?as=webp`.

For example in `index.blade.php`: `<img src="@asset('images/my-image.jpg?as=webp')" />`

### version

6.6.10

**Comment by kellymears** [comment]
the blade templates are not parsed with bud.js at all so there is no way for this to happen, currently. 

if we did set up html-loader to process blade files it would output the compiled blade templates to `public`, which is probably not ideal for most projects.

not sure about this, but re-tagging as a feature request since everything seems to be working as expected.

**Comment by kellymears** [comment]
One cool, kinda unexpected thing about the blade-loader is that it makes calling `bud.copy` or `bud.assets` unnecessary with a fresh copy of sage (since all the needed assets are included in the compilation directly now)

**Comment by kellymears** [comment]
This is included in [v6.7.0](https://github.com/roots/bud/releases/tag/v6.7.0). You need to add the blade files to an entrypoint in order for the `@asset` tag to be found by `@roots/blade-loader`.
